docs/develop/dialog-api-in-office-add-ins.md

```diff
@@ -288,6 +340,10 @@ See [Authenticate with the Office dialog API](auth-with-office-dialog-api.md).
 
 SPAs and client-side routing need to be handled with care when you are using the Office dialog API. Please see [Best practices for using the Office dialog API in an SPA](dialog-best-practices.md#best-practices-for-using-the-office-dialog-api-in-an-spa).
 
+### Message dialog from parent when messageChild is not available
+
+In some situations, the `messageChild` API may not be supported. Some alternative ways for parent to dialog box messaging are described in [Alternative ways of passing messages to a dialog box from its host page](parent-to-dialog.md).
```

Comment:
It feels weird to have this advice twice in the same article. Isn't the above note sufficient?